index.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { Provider } from "react-redux";
+import store from "./state/store/store";
+import { registerRootComponent } from "expo";
+import App from "./App";
+
+const ConnectedApp = () => (
```

Comment:
if you want this type of syntax use `return ()`

Reply:
Added it!